nlu/docker-compose.train.yml

```
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  ludwig:
+    image: ludwigai/ludwig:master
+    command: >
+      experiment --config /src/${MODEL_NAME}.yaml
+                 --dataset /data/data.json
+                 --output_directory /src/results
```

```suggestion
                 --output_directory /src/results
```
same for results ...